recap-project-4/src/Components/Form.js

```diff
@@ -3,23 +3,34 @@ import "./Form.css";
 export default function Form() {
   function handleSubmit(event) {
     event.preventDefault();
-    const form = event.target;
-    event.target.reset();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    console.log(data);
+
+    const isForGoodWeather = data.isForGoodWeather === "on" ? true : false;
```

Comment:
a comparison always returns `true` or `false`, which means you could write this line of code even more concise:

```js
const isForGoodWeather = data.isForGoodWeather === "on"
```